### PR TITLE
Update patch version of dependencies + Major faceted search module

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Update patch version of dependencies (using `composer update --patch-only`) + Major faceted search module
| Type?             | bug fix
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green and UI tests green
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/12240331543
| Fixed issue or discussion?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Comment by Hlavtox** [comment]
Thank you @jolelievre! :-)

**Pull request merged**
